src/Parser.zig

```diff
@@ -3344,7 +3344,7 @@ fn findScalarInitializer(p: *Parser, il: **InitList, ty: *Type, actual_ty: Type,
         }
         const struct_il = il.*;
         while (index < fields.len) : (index += 1) {
-            const field = fields[@intCast(u32, index)];
+            const field = fields[@as(u32, @intCast(index))];
```

Comment:
Why was there a cast here in the first place?

Reply:
This one might not actually be needed; I'll push an update and see if it still builds